docs/application/dotnet/guides/nui/xaml/xaml-support-for-tizen-nui.md

```diff
@@ -0,0 +1,715 @@
+
+![XAMLFile](./media/xaml-page-xaml-file.PNG)
+
+In this sample, the `TextLabel` element is added, and the `Text`, `PointSize`, `TextColor` and other properties are set.
+
+### Create C# Code-Behind File
+
+After creating the **NUIXamlTemplate** solution, you can see `XamlPage.xaml.cs`, which is a C# code-behind file associated with the XAML file in the project folder. 
+
+![XAMLCodeBehindFilePath](./media/xaml-page-xaml-cs-code-behind.PNG)
+
+> **Note**
+>
+> If you select other templates or work on lower version than Tizen 5.5, you have to create a C# code-behind file manually.
+
+In the following sample XAML code, the `x:Class` attribute specifies a fully qualified class name, the `XamlPage` class in the `NUIXamlTemplate` namespace. This means that this XAML file of **XamlPage.xaml.cs** defines a new class named `XamlPage` in the `NUIXamlTemplate` namespace that is derived from `ContentPage`:
+
+```csharp
+using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+
+namespace NUIXamlTemplate
+{
+    public partial class XamlPage : ContentPage
+    {
+        public XamlPage(Window win) : base(win)
+        {
+            InitializeComponent();
+        }
+    }
+}
+```
+
+> **Note**
+>
+> The name of the XAML file must be same as the class name.
+
+### Load XAML
+
+To load layout instance using XAML, create a `XamlPage` instance in your code such as `OnCreate()` method that is invoked when the application is started:
+
+```csharp
+using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+
+namespace NUIXamlTemplate
+{
+    class Program : NUIApplication
+    {
+        Animation animation;
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+            Window.Instance.BackgroundColor = Color.Blue;
+            Window.Instance.KeyEvent += OnKeyEvent;
+
+            XamlPage page = new XamlPage(Window.Instance);
+
+            animation = new Animation(2000);
+            animation.AnimateTo(page.text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, 500);
+            animation.AnimateTo(page.text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), 500, 1000);
+            animation.Looping = true;
+            animation.Play();
+        }
+
+        public void OnKeyEvent(object sender, Window.KeyEventArgs e)
+        {
+            if (e.Key.State == Key.StateType.Down && (e.Key.KeyPressedName == "XF86Back" || e.Key.KeyPressedName == "Escape"))
+            {
+                Exit();
+            }
+        }
+
+        static void Main(string[] args)
+        {
+            var app = new Program();
+            app.Run(args);
+        }
+    }
+}
+```
+
+When you compile and run this program, the `TextLabel` element will appear on the window. Also you can see the text of "Hello, NUI XAML APP", which is defined in xaml file, animates rotating on the x-axis.
+
+
+## Create Solution Before Tizen 5.5
```

Comment:
Suggestion: Could we give another heading?
 For example: Create Solution for Tizen Version 5.0 and Lower

@mijin85cho, please suggest.

Reply:
@surbhig12 I understand you tried to keep the consistency for the words, but how about using shorter words for the **title**?
For example like :

Create Solution Under Tizen 5.5

Reply:
@mijin85cho,
In that case, there is a confusion whether version 5.5 is included or not. To be more clear, I suggested "Tizen Version 5.0 and Lower".

However, I agree that the heading is a bit long but could we take this as an exception considering there would be no confusion with respect to versions?
Please correct me if I am wrong.

Reply:
@surbhig12 Thanks for sharing your concern.

I thought the "under" has the meaning of "less than" as like below link. So I used "under" as it is clear that Tizen 5.5 is not included in my thought.

https://dictionary.cambridge.org/ko/%EC%82%AC%EC%A0%84/%EC%98%81%EC%96%B4/under

![image](https://user-images.githubusercontent.com/9958374/73634167-e7281a00-46a3-11ea-9eca-8fd9e36f1418.png)


However as "Create Solution Under Tizen 5.5" is not tidy, the other form, you suggested can be considered. You are correct that I don't think long sentence is proper to the title, but for the clear meaning, we may can give up the simplicity.

Before we go with your suggestion, if you can give us another suggestion, please let me know.

Reply:
@mijin85cho, thank ou for the discussion.
The only best option to avoid confusion and ensure clear meaning  I can suggest:

Create Solution Using Tizen Version 5.0 and Lower